timemachine/fe/single_topology_v3.py

```diff
@@ -483,6 +484,211 @@ def find_dummy_groups_and_anchors(mol_a, mol_b, core_a, core_b):
     return dummy_groups_b, all_jks
 
 
+def handle_ring_opening_closing(
+    f: Callable[[float, float, float], float],
```

Comment:
Preference: type alias `InterpolationFxn`, more suggestive name `interpolate_fxn`, or docstring line indicating parameters of `f(src, dst, lamb) -> interpolated`

Reply:
Good call. I added the suggestion to the docstring in 861cc6d (coming up with a general type for `InterpolationFxn` at this stage seemed a bit complicated due to mixed conventions with passing scalars and arrays)